src/App.css

```diff
@@ -31,6 +31,19 @@
   border-bottom: 1px solid #505c66;
 }
 
+.sub-header-step-2 {
+  font-size: 2rem;
+  font-weight: 700;
+  font-family: 'Roboto Slab', sans-serif;
+  color: #b85a27;
```

Comment:
For colors, we store those as variables in `styleController.ts`. Could you add terracotta to this file, and then you can access it with `var(--variable-name)`. This allows us to add white labels where partners can use a different color pallet.